hack/bin/upload-images.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+# This script builds all the images in wireServer.images attribute of
+# $ROOT_DIR/nix/default.nix and uploads them to the docker registry using the
+# repository name specified in the image derivation and tag specified by
+# environment variable "$DOCKER_TAG".
+#
+# If $DOCKER_USER and $DOCKER_PASSWORD are provided, the script will use them to
+# upload the images.
+#
+# This script is intended to be run by CI/CD pipelines.
+
```

Comment:
This seems to do pretty much the same as upload-hooge-image. Can the scripts be unified, and expect the attributes / attribute to build as args?